app/build.gradle.kts

```
@@ -0,0 +1,173 @@
+        targetCompatibility = JavaVersion.VERSION_11
+        isCoreLibraryDesugaringEnabled = true
+    }
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11.toString()
+    }
+    buildFeatures {
+        compose = true // Enables Jetpack Compose for this module
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = Config.Compose.COMPOSE_COMPILER_VERSION
+    }
+    signingConfigs {
+        create("release") {
+            // Release keystore expected to be present in environment variables (living on the build server)
+            storeFile = file(System.getenv("_KEYSTORE") ?: "_KEYSTORE environment variable not set for release build type; unable to compile the current variant")
+            storePassword = System.getenv("_KEYSTORE_PASSWORD")
+            keyAlias = System.getenv("_KEY_ALIAS")
+            keyPassword = System.getenv("_KEY_PASSWORD")
+        }
+    }
+    // See BEST_PRACTICES.md for comments on purpose of each build type/flavor/variant
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = true
+            isShrinkResources = true
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+           //signingConfig = signingConfigs.getByName("release")
+        }
+        getByName("debug") {
+            applicationIdSuffix = ".debug"
+            versionNameSuffix = "-debug"
+            // Disabling as leaving it enabled can cause the build to hang at the jacocoDebug task for 5+ minutes with no observed adverse effects when executing
+            // the test...UnitTestCoverage tasks. Stopping and restarting build would allow compilation/installation to complete.
+            // Disable suggestion found at https://github.com/opendatakit/collect/issues/3262#issuecomment-546815946
+            isTestCoverageEnabled = false
+        }
+        // Create debug minified buildtype to allow attaching debugger to minified build: https://medium.com/androiddevelopers/practical-proguard-rules-examples-5640a3907dc9
+        create("debugMini") {
+            initWith(getByName("debug"))
+            matchingFallbacks += listOf("debug")
+            isMinifyEnabled = true
+            isShrinkResources = true
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        }
+    }
+    flavorDimensions += listOf("environment")
+    // See BEST_PRACTICES.md for comments on purpose of each build type/flavor/variant
+    productFlavors {
+        create("internal") {
+            applicationId = "com.bottlerocketstudios.mapsdemo.internal"
+            versionNameSuffix = "-internal"
+            buildConfigField("boolean", "INTERNAL", "true")
+            buildConfigField("boolean", "PRODUCTION", "false")
+            dimension = "environment"
+        }
+        create("production") {
+            applicationId = "com.bottlerocketstudios.mapsdemo.internal"
```

@Josh-Gebbeken Remove `.internal` from the production applicationId

```suggestion
            applicationId = "com.bottlerocketstudios.mapsdemo"
```